fix: Can't delete datasource error (#1749)
appsmithorg/appsmith-docs: website/docs/help-and-support/troubleshooting-guide/action-errors/datasource-errors.md

> ### Can't delete datasource error

Reviewer: Update the title to use Cannot-
```suggestion
### Cannot delete datasource error
```

> If you are trying to delete a [datasource](/connect-data/reference/) and encounter an error message similar to:

Reviewer: Rewrite this - 
```suggestion
You may face this issue when deleting a [datasource](/connect-data/reference/).
```
